src/prommis/uky/costing/ree_plant_capcost.py

```diff
@@ -1946,7 +1946,7 @@ def taxes_and_insurance_cost_rule(c):
         @b.Constraint()
         def sum_watertap_fixed_cost(c):
             if not hasattr(c, "watertap_fixed_costs_list"):
-                return c.watertap_fixed_costs == 0
+                return c.watertap_fixed_costs == 1e-12
```

Comment:
Why are you changing this from 0 to such a small value?

Reply:
The diagnostics indicated an error from fixing the small value - the model can still solve with this specification.

Reply:
Was that an error, or just a caution?  This looks like a case where 0 should be a valid value, and thus you probably don't need to change this. The exception might be a possible evaluation error, but if that is the case it is more likely to be due to another unbounded variable.

Reply:
You're correct, the check fails because those variables have lower bounds at 0. As these variables will always be fixed to 0 or a user input value, we could just remove the bounds and fix them to zero.